.github/workflows/appsignals-python-e2e-test.yml

```diff
@@ -31,20 +31,31 @@ env:
   APP_SIGNALS_PYTHON_E2E_RE_SA_IMG: appsignals-python-django-remote-service
   METRIC_NAMESPACE: AppSignals
   LOG_GROUP: /aws/appsignals/eks
-#  ECR_OPERATOR_STAGING_REPO: ${{ vars.ECR_OPERATOR_STAGING_REPO }}
+  ECR_OPERATOR_STAGING_REPO: ${{ vars.ECR_OPERATOR_STAGING_REPO }}
 
 
 jobs:
   appsignals-e2e-test:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
+      - name: Download enablement script
+        uses: actions/checkout@v4
         with:
-          # TODO: This step is used for custom pre-release instrumentation
-          # Uisng SHA as of March 4. It is a temporary measure until the cw add-on is released with python support
-          ref: abf75babe672412cb63c56cbcf1c5ce2d8c97a1c
-          fetch-depth: 0
+          repository: aws-observability/application-signals-demo
+          ref: main
+          path: enablement-script
+          sparse-checkout: |
+            scripts/eks/appsignals/enable-app-signals.sh
+            scripts/eks/appsignals/clean-app-signals.sh
+          sparse-checkout-cone-mode: false
+
+      - name: Use latest addon version
+        working-directory: enablement-script/scripts/eks/appsignals
+        run: |
+          sed -i 's/aws eks create-addon \\/aws eks create-addon \\\n        --resolve-conflicts OVERWRITE \\/' enable-app-signals.sh
+          sed -i 's/if \[\[ "$addon_version" < "v1\.2\.0" \]\]/if [[ "$addon_version" < "v1.4.0" ]]/' enable-app-signals.sh
+          sed -i 's/--addon-version v1\.2\.0-eksbuild\.1/--addon-version v1.4.0-eksbuild.1/' enable-app-signals.sh
+          cat enable-app-signals.sh
```

Comment:
Same comment as in https://github.com/aws-observability/aws-otel-python-instrumentation/pull/124

Reply:
Updated.